src/overlaybd/fs/test/CMakeLists.txt

```diff
@@ -4,21 +4,7 @@ link_directories($ENV{GFLAGS}/lib)
 include_directories($ENV{GTEST}/googletest/include)
 link_directories($ENV{GTEST}/lib)
 
-add_executable(fs_test test.cpp)
```

Comment:
can these tests be kept?

Reply:
I'll test and re-organize these code later